### ISSUE TITLE
Nuget package

### ISSUE THREAD

**Issue by meriturva** [Danielku15/OpenApiQuery#41]
Just to know, are you going to deploy the library on NuGet?
I guess when it will be online it could have more interest from the community!

**Comment by Pzixel** [comment]
Hello. I would defenitely like to have this as a nuget package

**Comment by Danielku15** [comment]
My main driver to work on this project was the need in some project at work. Due to some shifts in our focus there is currently no urgent need for replacement and I currently do not really have the time to work on this library to get it really production ready.

I'm happy to accept contributions to get it to a proper state but I fear there is still a lot to do to really be suitable for production ready scenarios. Setting up a CI/CD pipeline with nuget deploy should be a matter of a few hours but I doubt that the current state is really satisfying the needs of the market.

**Comment by Pzixel** [comment]
Yes, I'm trying to get it work with my mongodb database, and it's an experiment on its own. I think I will try to send some PRs if I find how to fix them. RIght now I've just forked this repo, because I need massive amount of features so implement (such as `select` in `expand` or true async materialization instead of fake one). 

I'm not sure if we choose this project, because I have multiple issues with mongodb handling IQueryable requests, but I still hope I will be able to. If so I will try to merge whatever suitable.
